frontend/src/lib/k8s/api/v2/utils.ts

```diff
@@ -0,0 +1,12 @@
+export function makeUrl(urlParts: any[], query: Record<string, string> = {}) {
```

Comment:
Is there a better place this can go than "utils"?
(which is a bit of an antipattern)

Reply:
other stuff in the file wasn't used and only makeUrl left, renamed to makeUrl